requirements.txt

```diff
@@ -31,67 +44,80 @@ Flask-Script==2.0.6
 Flask-SQLAlchemy==2.5.1
 Flask-Versioned==0.9.4.post20101221
 future==0.18.2; python_version < '3.0'
-google-auth==1.35.0; python_version < '3.0'
 google-auth==1.23.0; python_version == '3.5'
-google-auth==2.1.0; python_version > '3.5'
-huey==2.3.2
+google-auth==2.6.6; python_version != '3.5'
+huey==2.4.3
 idna==2.10; python_version <= '3.5'
-idna==3.2; python_version > '3.5'
-importlib_metadata==2.1.1; python_version <= '3.5'
-importlib_metadata==4.6.0; python_version > '3.5'
+idna==3.3; python_version > '3.5'
+importlib-metadata==2.1.3; python_version <= '3.5'
+importlib-metadata==4.8.3; python_version == '3.6'
+importlib-metadata==4.11.3; python_version > '3.6'
 importlib-resources==3.3.1; python_version < '3.0'
 importlib-resources==3.2.1; python_version == '3.5'
+importlib-resources==5.4.0; python_version == '3.6'
 ipaddress==1.0.23; python_version < '3.0'
 itsdangerous==1.1.0
 Jinja2==2.11.3
-ldap3==2.9
-lxml==4.6.5
-Mako==1.1.4
+ldap3==2.9.1
+lxml==4.8.0
+Mako==1.1.6; python_version <= '3.6'
+Mako==1.2.0; python_version > '3.6'
 MarkupSafe==1.1.1; python_version <= '3.5'
 MarkupSafe==2.0.1; python_version > '3.5'
 netaddr==0.8.0
+packaging==20.9; python_version < '3.0'
+packaging==21.3; python_version > '3.5'
 passlib==1.7.4
-pathlib2==2.3.6; python_version <= '3.5'
+pathlib2==2.3.7.post1; python_version < '3.0'
 Pillow==6.2.2; python_version < '3.0'
 Pillow==7.2.0; python_version == '3.5'
 Pillow==8.4.0; python_version == '3.6'
-Pillow==9.0.1; python_version > '3.6'
+Pillow==9.1.0; python_version > '3.6'
 pyasn1==0.4.8
 pyasn1-modules==0.2.8
-pycparser==2.20
+pycparser==2.21
 pydash==4.9.3; python_version <= '3.5'
-pydash==5.0.1; python_version > '3.5'
+pydash==5.1.0; python_version > '3.5'
 PyJWT==1.7.1; python_version <= '3.5'
-PyJWT==2.1.0; python_version > '3.5'
+PyJWT==2.3.0; python_version > '3.5'
 PyMySQL==0.10.1; python_version <= '3.5'
 PyMySQL==1.0.2; python_version > '3.5'
-pyOpenSSL==20.0.1
+pyOpenSSL==21.0.0; python_version < '3.0'
+pyOpenSSL==20.0.1; python_version == '3.5'
+pyOpenSSL==22.0.0; python_version > '3.5'
+pyparsing==3.0.8; python_version > '3.5'
 pyrad==2.4
-python-dateutil==2.8.1
-python-editor==1.0.4
-python-gnupg==0.4.7
-pytz==2021.1
+python-dateutil==2.8.2
+python-editor==1.0.4; python_version <= '3.5'
+python-gnupg==0.4.8
+pytz==2022.1
+PyYAML==5.4.1; python_version < '3.0'
```

Comment:
How crappy is 3.5?
Why do older versions of python allow a higher pyYAML version?